design-editor/src/utils/utils.js

```diff
@@ -69,10 +69,12 @@ module.exports = {
 	 * @param {object} bracketsEditor
```

Comment:
Please, correct documentation according to the current function state

Reply:
fixed